wurst/util/PlayerUtils.wurst

```diff
@@ -0,0 +1,104 @@
+package PlayerUtils
+import LinkedList
+
+/**
+*   Credits to TriggerHappy for making PlayerUtils [vJass]
+*    - Link : https://www.hiveworkshop.com/threads/playerutils.278559/
+**/
+
+User array USERS
+
+public function player.toUser() returns User
+    return USERS[this.getId()]
+
+public class User
+    static constant let activeUsers = new LinkedList<User>
+    static constant player local = GetLocalPlayer()
```

Comment:
already exists as localPlayer

Reply:
Comes side-by-side with localId, if you want I'll just erase them both.

Reply:
I suppose localId *could* be added to the Player package

Reply:
Actually looking at the code again, Local and LocalId is used internally to fetch a "User" / PlayerData object from the array so the user can do ClassName.fromLocal() to get their own Data Object from their Local ID. Could change it to use the variables from Player but I think it'd be nicer in code to keep available in the class.

But it's up to you guys how you want it in the end.